### DISCUSSION TITLE
Solo farming or joining forces with my brother. Will it increase or decrease our win chances?

### DISCUSSION THREAD

**Discussion by cayolblake** [Chia-Network/chia-blockchain#5586] [Q&A]
Hello,

Got a question for experts, my brother and I are pooling/farming separately, around the same numbers, if we merge our forces is our chances higher in making a win or each separate?

The main point is will this increase or decrease our winning chances.

Much appreciate explanations as to why either or.

Many thanks.

**Comment by climba03003** [comment]
Winning chance is the same.

**Reply by cayolblake** [reply]
How so? Can you please provide a proof on that?

Isn't each wallet/farmer expected to have 4608 chances of winning a day? If we double on that we are multiplying by two? 🤔

**Reply by climba03003** [reply]
No, your plots is passing the plot filter with the same chance. So, you have the same chance joining together or split.

**Reply by cayolblake** [reply]
What if we replicated our environment into 2 identical setups with a single wallet/farmer/node and both connected to the same pool of harvesters. Would that make any difference?

**Reply by climba03003** [reply]
Simplified example:
- It do not include the challenge parts, so it assume all plots with the same size (K32)

### Spliting
2 farmers with 100 plots each = 200 plots in total
plots of passing plot filter a day will be
200 / 512 * 9216 = 3600
that 3600 times is the number of chances you will be able to win.

### Joining
1 farmers with 200 plots = 200 plots in total
plots of passing plot filter a day will be
200 / 512 * 9216 = 3600

Both numbers of times passing the plot filter is the same

**Reply by climba03003** [reply]
Increase number of plots (means increase space) is the only way to increase the chances to win.

**Reply by cayolblake** [reply]
Okay I see.

But, If we got 10 machines each with its own IP address only running wallet/farmer/full_node and all connected to a single harvester, will this still not count? 

I assume this cause we will have more presence on Chia (from a networking perspective)! 🤔

**Reply by climba03003** [reply]
> 
> 
> Okay I see.
> 
> But, If we got 10 machines each with its own IP address only running wallet/farmer/full_node and all connected to a single harvester, will this still not count?
> 
> I assume this cause we will have more presence on Chia (from a networking perspective)! 🤔

It do not count as all plots have its own `plot id`. They will be passing the plot filter at the same signage points.
No matter how many fullnode or farmer using the same plot, you will only get 2XCH if it won (challenge computing will have the same result because the file data is the same).

**Reply by cayolblake** [reply]
Okay I see. 

If that's the case in general, then why/how un-official pools claim to provide more consistent/guaranteed profits?

Is it because they theoretically enter the lottery with more plots?

**Reply by climba03003** [reply]
Pools with more plots have high chance to win. People like to join the pool with highest share, but the outcome will be the reward will greatly split-ed. More importantly, the total number of plots for a pools will be increased by anyone who join it (it makes the winning chance increase.)

In your case, the total numbers of plots of is the same, so no matter you are in pool or solo farm the chance is the same.

**Comment by cayolblake** [comment]
Final question,

Does this make http://chiacalculator.com kind of scam/optimistic?

I've read in the docs how to set the expectations and possibilities of winning, but, let's say I want to hit a %90 guarantee to make a daily winning, how many plots I "should" be "at least" having given the current complexity and network size?

**Reply by climba03003** [reply]
It is hard to calculate until the network growth become stable

**Reply by cayolblake** [reply]
Ah but there's a hard/soft limit? 

So one can say if you have a 1,000 plots, you can win in a day but not later than a week?

Also, regarding my previous network related point, I understand multiple plots can contain the same "so called lottery ticket", so if I have more presence on the network then I increase my chances of a win? Since I'll be having more than one IP address so increasing possibility of getting asked for the lottery ticket value. Correct?

**Reply by climba03003** [reply]
Not true for your point

Signage Point is the same hashes for over all network.

It's like a bingo game, your Plot is a bingo sheet. It will be the same Bingo sheet when it appear in the network. 
No matter how many times it reach the same signage point (bingo words). The result is the same.

It is only wasting resources for creating multiple fullnode.
